app/assets/javascripts/qpixel_api.js

```diff
@@ -7,12 +7,6 @@ const validators = [];
 let popped_modals_ct = 0;
 
 window.QPixel = {
-  csrfToken: () => {
```

Comment:
Since CSRF token is now sent as a cookie, there's no longer a need to read the token from DOM or add it as a `X-CSRF-Token` header for that matter.